carma-messenger-ui/website/widgets/emergencyResponse/widget.js

```diff
@@ -193,6 +246,48 @@ var goToEventManagement = () => {
     CarmaJS.WidgetFramework.closeEventManagementWidgets();
     CarmaJS.WidgetFramework.loadEventManagementWidgets();
 }
+/***
+ * Load OpenStreetMap with MapboxGL JS
+ */
+var loadMap = () => {
+    mapboxgl.accessToken = "pk.eyJ1IjoiZGR1MjAyMCIsImEiOiJjbDJyeHJob2YwYnhwM2xtaG9zaDdnYTR4In0.Rh2bSS44c99BoDj2W7jjfw";
+    let default_center = [-77.150495, 38.955675];
```

Comment:
Let's add a note here to state that TFHRC has been selected as the default center since it it where the STOL development team is located.

Reply:
https://github.com/usdot-fhwa-stol/carma-messenger/pull/177/commits/1ba7fd9440d4eed97e1d767b735e1315afdfa5d5